slacker.go

```diff
@@ -363,14 +380,22 @@ func (s *Slacker) handleMessageEvent(ctx context.Context, evt interface{}) {
 	}
 }
 
-func newMessageEvent(evt interface{}) *MessageEvent {
+func newMessageEvent(slacker *Slacker, evt interface{}, req *socketmode.Request) *MessageEvent {
 	var me *MessageEvent
 
 	switch ev := evt.(type) {
 	case *slackevents.MessageEvent:
 		me = &MessageEvent{
-			Channel:         ev.Channel,
-			User:            ev.User,
+			Channel: ev.Channel,
+			ChannelName: func() string {
```

Comment:
This an be changed to `ChannelName: string` but then this field will be not lazy fetched anymore.
Instead the call to `channel, _ := slacker.client.GetConversationInfo(ev.Channel, true)` has to be performed
during the creation of the `MessageEvent`.

If this is okay then i will chaange it.

Reply:
You're right, and generally I would side on you with doing lazy fetching to save a few http calls. However in this case its not clear to me that the performance saved here is worth breaking consistency with the rest of the fields, and I can see someone using this getting tripped up, so I'm electing to keep things consistent.

If performance becomes an issue we can definitely revisit this.